lic-nz/allflex-product-mapper
# Add validation to xml

* Add Nokogiri gem
* Add validation to XMLFormatter
* Add schema to test against
* Handle validation result in XmlSplittingTypeFormatter
* Add spec to test the failure
* Update manufacturing definition and resultant xml as it didn't confirm
